### ISSUE TITLE
Suseptible shows negative values in SIR-FV model

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#157]
## Summary
The number of cases must a natural number, but Susceptible shows negative values in SIR-FV model.

## Related classes
- `covsirphy.SIRFV`

## Codes and outputs:
```Python
import covsirphy as cs
# Dataset preparation
population_data = cs.PopulationData(filename=None)
example_data = cs.ExampleData(tau=1440, start_date="01Jan2020")
preset_dict = cs.SIRF.EXAMPLE["param_dict"]
area = {"country": "Theoretical"}
# Create dataset from 01Jan2020 to 31Jan2020
example_data.add(cs.SIRF, step_n=30, **area)
# Register population value
population_data.update(cs.SIRF.EXAMPLE["population"], **area)
snl = cs.Scenario(example_data, population_data, tau=1440, **area)
snl.add(end_date="31Jan2020", model=cs.SIRF, **preset_dict)
# Add vaccine scenario
snl.add(end_date="31Dec2020", name="Vaccine", model=cs.SIRFV, omega=0.001)
df = snl.simulate(name="Vaccine", y0_dict={"Vaccinated": 0})
```
In this dataframe, records Susceptible < 0 are included.

## Environment
- CovsirPhy version: 2.6.0-beta
- Python version; 3.8
- Installation: pipenv
- OS: WSL

**Comment by lisphilar** [comment]
Preset of SIR-FV parameter will be changed.
Omega: from 0.001 to 0.01

**Comment by lisphilar** [comment]
Expression of SIR_FV day-paramerters will be fixed.
- "alpha" to "alpha [-]"
- "Vaccinated [persons]" to "Vaccinated [persons/day]"

**Comment by lisphilar** [comment]
This was fixed in 2.6.0-gamma.fix.120.155.157
